package.json

```diff
@@ -15,8 +15,8 @@
 	"scripts": {
 		"start": "gulp watch",
 		"compile": "gulp",
-		"check-formatting": "prettier --check .",
-		"format": "prettier --write ."
+		"check-formatting": "prettier --check scripts/ styles/",
+		"format": "prettier --write scripts/ styles/"
```

Comment:
Because at some point the prettier was whining about code in `/node_modules`, `/deps` and `/modules`. The prettier should only check our own code, not others people code which we use as a dependency. Moreover `.` also checks the files on the rool level such as `package-lock.json` which is an auto-generated file by `npm update`.

Moreover, efficiency. The prettier should not even parse the whole `/node_modules` with thousands of files even if it does not find anything.